be/src/storage/tablet_updates.cpp

```diff
@@ -475,6 +479,8 @@ Status TabletUpdates::rowset_commit(int64_t version, const RowsetSharedPtr& rows
 }
 
 Status TabletUpdates::_rowset_commit_unlocked(int64_t version, const RowsetSharedPtr& rowset) {
+    // optimization: if rowset is empty, do not add it to rowset collection
+    bool rowset_empty = rowset->num_segments() + rowset->num_delete_files() == 0;
```

Comment:
E0 = rowset->num_segments()
E1 = rowset->num_delete_files()
if use E0 + E1 == 0 to check if E0 == 0 && E1 == 0, it's correctly arithmetically
but it is not a logical equivalence, for example, E0 == -1 && E1 == 1, it make the E0 + E1 == 0 satisfied as well,
so reader of this code snippet will be confused: is there a certain rule/significance of this equivalent relationship.
maybe use E0 == 0 && E1 == 0 is more readable and clearly.

Reply:
will add a method in rowset